content/en/monitors/create/types/ci_pipelines.md

```diff
@@ -0,0 +1,119 @@
+---
+title: CI Pipeline Monitor
+kind: documentation
+aliases:
+- /monitors/monitor_types/ci_pipelines
+further_reading:
+- link: "/monitors/notify/"
+  tag: "Documentation"
+  text: "Configure your monitor notifications"
+- link: "/monitors/notify/downtimes/"
+  tag: "Documentation"
+  text: "Schedule a downtime to mute a monitor"
+- link: "/monitors/manage/status/"
+  tag: "Documentation"
+  text: "Check your monitor status"
+---
+
+## Overview
+
+Once [CI Visibility is enabled][1] for your organization, you can create a CI Pipeline monitor to alert you when a specified type of CI Pipeline event exceeds a user-defined threshold over a given period of time.
+
+<div class="alert alert-info"><strong>Note</strong>: Only available in closed beta on US1, EU, US3 and US5.</div>
+
+## Monitor creation
+
+To create a [CI Pipeline monitor][2] in Datadog, use the main navigation: *Monitors -> New Monitor --> CI Pipelines*.
+
+<div class="alert alert-info"><strong>Note</strong>: There is a default limit of 1000 CI Pipeline monitors per account. <a href="/help/">Contact Support</a> to lift this limit for your account.</div>
+
+### Define the search query
+
+1. Construct a search query using the same logic as a CI Pipeline explorer search.
+2. Select the CI Pipeline events level to monitor:
+    * **Monitor over the `Pipeline` level**: If the `Pipeline` level is selected, the monitor will check the events that
```

Comment:
I think it misses the end of the sentence

Reply:
fixed